watertap/edb/data_model.py

```diff
@@ -862,6 +875,21 @@ def __init__(self, data: Dict, validation=True):
     def reaction_type(self):
         return self.data.get("type", "")
 
+    # # TODO: This function behavior is WRONG!!!
```

Comment:
@aladshaw3 is this something that should be addressed now or reserved for a later PR?

Reply:
This is going to be resolved in Dan's next PR